sdk/communication/azure-communication-sms/README.md

```diff
@@ -128,15 +128,32 @@ for (SmsSendResult result : sendResults) {
 
 ## Troubleshooting
 
-All SMS service operations will throw an exception on failure.
-<!-- embedme src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java#L104-L112 -->
+- SMS operations will throw an exception if the request to the server fails.
+- Exceptions will not be thrown if the error is caused by an individual message, only if something fails with the overall request.
+- Please use the `isSuccessful()` flag to validate each individual result to verify if the message was sent.
+<!-- embedme src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java#L153-L176 -->
 ```java
 try {
-    SmsSendResult sendResult = smsClient.send(
+    SmsSendOptions options = new SmsSendOptions();
+    options.setDeliveryReportEnabled(true);
+    options.setTag("Marketing");
+
+    Response<Iterable<SmsSendResult>> sendResults = smsClient.sendWithResponse(
         "<from-phone-number>",
-        "<to-phone-number>",
-        "Hi"
-    );
+        Arrays.asList("<to-phone-number1>", "<to-phone-number2>"),
+        "Hi",
```

Comment:
Can we update the message to "Weekly Promotion" so that it correlates with the tag? Refer to the C# README as an example.